core/market/decentralized/tests/test_initial_proposal.rs

```diff
@@ -22,7 +23,7 @@ async fn test_query_events_non_existent_subscription() -> Result<(), anyhow::Err
     let non_existent_id = "80da375cb604426fb6cddd64f4ccc715-85fdde1924371f4a3a412748f61e5b941c500ea69a55a5135b886a2bffcb8e55".parse()?;
 
     // We expect that no events are available for non existent subscription.
-    let result = market1.query_events(&non_existent_id, 1.2, Some(5)).await;
+    let result = market1.query_events(&non_existent_id, 0.2, Some(5)).await;
```

Comment:
This comment applies to all timeout changes:
Are you sure, you want to decrease them???
Note that in normal execution that doesn't hurt us at all, because we use notification to wake up from waiting.
In incorrect execution it matters, but I think it's good to have bigger margin, to avoid non deterministic tests.

Reply:
I've decided to give it a try, but your comment given me opportunity to think again and I would not mix changing test execution (with macro) and other changes. This will enable easier reverting in case of failure of this approach with `serial_test`.